docker/el7/Dockerfile

```
@@ -40,8 +68,19 @@ RUN cd /tmp && wget https://archive.mozilla.org/pub/opus/opus-1.3.1.tar.gz \
     && make -j5 \
     && make install
 
+# Restore old GCC's ld to build glibc
+RUN ln -sf /usr/bin/ld /usr/local/bin/ld
+
 RUN cd /tmp && wget http://ftp.gnu.org/gnu/glibc/glibc-2.18.tar.gz && tar zxvf glibc-2.18.tar.gz \
     && cd /tmp/glibc-2.18 && mkdir -p build && cd build \
-    && CC= CXX= LD_LIBRARY_PATH= ../configure --prefix=/opt/glibc-2.18 \
+    && CC=gcc CXX=gcc++ LD_LIBRARY_PATH= ../configure --prefix=/opt/glibc-2.18 \
```

I think `gcc++` does not exist (probably is not a problem because glibc is not a `C++` project. The GNU C++ compiler is `g++`

```suggestion
    && CC=gcc CXX=g++ LD_LIBRARY_PATH= ../configure --prefix=/opt/glibc-2.18 \
```